dll/win32/setupapi/queue.c

```diff
@@ -20,10 +20,14 @@
 
 #include "setupapi_private.h"
 
+#ifdef __REACTOS__
 #include <aclapi.h>
+#endif
 
 /* Unicode constants */
+#ifdef __REACTOS__
 static const WCHAR DotSecurity[]     = {'.','S','e','c','u','r','i','t','y',0};
```

Comment:
Why not `L".Security"`? Even Wine allows this now and this is inside a ROS ifdef.

Reply:
Yes, please don't review yet; it's being re-rebased (I'm chasing a build error). Iirc this is fixed in later commits.